plugins/dragonframe/plugin.json

```
@@ -4,7 +4,10 @@
   "name": "Dragonframe",
   "description": [
     "<p><em>Dragonframe</em> is software for stop motion filmmaking, with DMX programming capabilities.",
-    "Dragonframe 5 will support lighting fixtures.</p>"
+    "Dragonframe 5 will support lighting fixtures."
```

Trailing comma missing.

```suggestion
    "Dragonframe 5 will support lighting fixtures.",
```